lib/commands/device/local-mode.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license
+ * Copyright 2021 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import type { IArg } from '@oclif/parser/lib/args';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { tryAsInteger } from '../../utils/validation';
+
+interface FlagsDef {
+	enable: boolean;
+	disable: boolean;
+	status: boolean;
+	help?: void;
+}
+
+interface ArgsDef {
+	uuid: string;
```

Comment:
Would it be `string | number` ? Because of `parse: (dev) => tryAsInteger(dev)`
Where `params.uuid` is used in the code, I think that currently `tsc` is type checking against `string` only, but it could be `number` as well, so potential bugs are not being caught by `tsc`.

Reply:
I seem to recall there was some reason we didn't do this, but I can't remember exactly, will check.

Reply:
Ah it was flags where that caused a bit of an issue - ok in args.